R/TreeCluster-class.R

```diff
@@ -131,7 +131,8 @@ ClusterHierarchy <- function(hierarchy, col_regex=NULL, columns =NULL) {
     "ClusterHierarchy",
     hierarchy
   )
-  
+  #print(hierarchy)
+  hierarchy
   # tree <- TreeIndex(hierarchy)
   # rownames(tree) <- rownames(hierarchy)
   # 
```

Comment:
@kzintas why do we need this change ?

Reply:
I think I just wanted to make sure I am returning the correct object

Reply:
We don't need that